### ISSUE TITLE
DzN - doťahovanie datumu odoslania

### ISSUE THREAD

**Issue by ZdenkoPek** [bratislava/konto.bratislava.sk#1688]
Doťahovanie dátumu odoslania rozhodnutia potrebujeme pre lepšie fungovanie nastavenia sposobu dorucenia.

**Comment by erehulka** [comment]
Je v Norise tento dátum právoplatnosti rozhodnutia? Ak áno 0.5MD, ale predpokladám že snáď áno :D 
Treba dotiahnuť tento dátum len pre nové dane, alebo chceme ho pridať aj k tým starým? Ak aj k starým + 1MD
Ak za tým je aj to posielanie mailov + zobrazovanie tak na to pls vytvor nový task @ZdenkoPek

**Comment by ZdenkoPek** [comment]
> Je v Norise tento dátum právoplatnosti rozhodnutia? Ak áno 1MD, ale predpokladám že snáď áno :D Treba dotiahnuť tento dátum len pre nové dane, alebo chceme ho pridať aj k tým starým? Ak aj k starým + 1MD Ak za tým je aj to posielanie mailov + zobrazovanie tak na to pls vytvor nový task @ZdenkoPek

Ano, je v Norise. Ja by som to chcel pouzivat pre dve veci. Jedna je odosielanie mailov z bloomreach, do ktoreho by sme tento datum mali dotahovat. Druhy case je na FE, ze rozhodnutie im bolo odoslane, to vsak teraz neriesime. Dolezite je mat v BE tento datum a vediet ho pouzivat. Predpokladam, ze do Bloomreach si bude vediet @Balros tento datum dostat.

**Comment by erehulka** [comment]
Okej, chceme to aj dotiahnuť pre staré dane čo už máme v konte?

**Comment by ZdenkoPek** [comment]
Datum pravoplatnosti rozhodnutia - generuje sa v Norise az po doruceni danovnikovi t.j. po prevzati zasielky alebo po fikcii dorucenia (15 dni od dorucenia a neprevzatia). Na zaklade tejto informacie sa dopocita datum ravoplatnosti rozhodnutia, ktory datum dorucenia + 30 dni. Dokym tento datum nemame zobrazime dan v konte bez datumov s nejakou generickou hlaskou - musime premysliet a nadizajnovat.

**Comment by erehulka** [comment]
Dobre, takže my nič nerátame, len skúšame dotiahnuť tento dátum z Norisu ak ho daň ešte nemá. Kým ho dotiahneme a kým teda nenastane ten dátum tak daň neukážeme (toto ale bude treba spraviť nový task pre frontent) @ZdenkoPek
